app/js/reducers/discontinueOrderReducer.js

```diff
@@ -0,0 +1,31 @@
+import {
+  DISCONTINUE_ORDER_FAILURE,
+  DISCONTINUE_ORDER_SUCCESS,
+  DISCONTINUE_ORDER,
```

Comment:
@zeze111 I haven't seen added under actionTypes or used anywhere.

Reply:
thanks, must have missed it when I was removing it from my implementation